.github/workflows/build.yml

```diff
@@ -25,6 +25,12 @@ jobs:
       run: |
         npm install
         npm run compile
+    - name: Smoke test (Linux)
+      run: xvfb-run -a npm run test
```

Comment:
Why did you use `xvfb-run` ? Couldn't you just execute `npm run test` in all platforms?

Reply:
💡☕️ Thank you @lstocchi,  It is interesting because the latest Ubuntu fails with `code ELIFECYCLE`, which I figured because reasons why `vscode-kubernetes` extension uses same code https://github.com/Azure/vscode-kubernetes-tools/blob/master/.github/workflows/build-dev.yml#L38 

* A sample of failed tests Ubuntu-Latest with `npm run test` [here](https://github.com/Azure/vscode-aks-tools/runs/4586936249?check_suite_focus=true). 
* Borrowed context which works: https://github.com/Azure/vscode-kubernetes-tools/blob/master/.github/workflows/build-dev.yml#L38

Reply:
Thanks for trying @Tatsinnit ... not sure about it failing, i'll dig on it when i'll have some time. Good to merge for me. :+1: